dalle_flow_glid3/sample.py

```diff
@@ -95,29 +98,41 @@ def set_requires_grad(model, value):
 bert.half().eval()
 set_requires_grad(bert, False)
 
-from clip_client import Client
+blank_clip_embedding = None
+blank_bert_embedding = None
 
+async def do_run(runtime_args, text_emb_clip):
+    global blank_clip_embedding, blank_bert_embedding
 
-async def do_run(runtime_args):
     if runtime_args.seed >= 0:
         torch.manual_seed(runtime_args.seed)
 
+    if blank_bert_embedding is None:
+        print('Generating blank BERT embedding')
+        blank_bert_embedding = (
+            bert.encode([runtime_args.negative])
+        ).to(device).float()
+
+    if blank_clip_embedding is None:
+        print('Generating blank CLIP embedding')
+        res = requests.post('https://demo-cas.jina.ai:8443/post',
+            data=json.dumps({'execEndpoint':'/', 'data': [{'text': runtime_args.negative}]}),
+            headers={'content-type': 'application/json'}
+        ).json()
+
+        blank_clip_embedding = np.array(res['data'][0]['embedding']).astype(
+            model_config['use_fp16'] and np.float16 or np.float32
+        ).reshape(1, -1)
```

Comment:
why not use grpc via cas client? What's more, the endpoints of cas-server would have some broken changes (to support multi-models by a single flow). So I recommend to use the cas-client instead.